### PR TITLE
Customize new templates

### PR DESCRIPTION
**Pull request by nathanbegbie**
search_results.pt, search_box.pt, paginator.pt and 404.pt

### CODE REVIEW

**Comment by nathanbegbie** [comment]
mama_mobile_community___week-by-week_guide_to_pregnancy_and_infants

mama_mobile_community___week-by-week_guide_to_pregnancy_and_infants

@SteveBarnett can we pls chat later regarding display of search results?

File: unicorecmsmama/static/css/style.css [hunk 1]
```diff
@@ -86,13 +86,18 @@
...
-
+/* SEARCH RESULTS*/
+	.search-results { padding: 14px 16px; margin: 0; border: 0; border-bottom: 1px solid #e6e6e6;}
 /*  MENU  */
 
 	#menu { padding: 10px 16px; background: #f7a11a; color: #ffc20f; text-align: left; font-weight: bold; }
 	[dir='rtl'] #menu { padding: 10px 16px; background: #f7a11a; color: #ffc20f; text-align: right; font-weight: bold; }
 		#menu a, #menu span { color: #fff; }
 
+/* BASE */
```
Comment (SteveBarnett) [inline unicorecmsmama/static/css/style.css]: Pull this and `SEARCH RESULTS` together into `SEARCH`

**Comment by SteveBarnett** [comment]
On the search results page, please add value="${query}" on the search input.

People will often refine a query, or try something similar if they don't find what they were looking for a first pass.

**Comment by SteveBarnett** [comment]
@nathanbegbie search results look good to me. Still want to chat?

Otherwise :+1:

**Comment by nathanbegbie** [comment]
@SteveBarnett Nah, if you're ok with them, that's fine. Just wasn't sure about the link colour.

**Comment by SteveBarnett** [comment]
@nathanbegbie re links: ah, I see what you mean. Good catch.
Please wrap the links in a `span` with class `.category-{{category-name}}` so that they are like the category names on the home page, under recommended.

**Comment by nathanbegbie** [comment]
@miltontony Please Review

**Comment by nathanbegbie** [comment]
mama_mobile_community___week-by-week_guide_to_pregnancy_and_infants
@SteveBarnett are you feeling the colours, or no?

**Comment by nathanbegbie** [comment]
mama_mobile_community___week-by-week_guide_to_pregnancy_and_infants
or this as an alternative? Maybe with bigger text, but that means fiddling with the CSS.

**Comment by miltontony** [comment]
@nathanbegbie I prefer the 2nd option here

**Comment by SteveBarnett** [comment]
@nathanbegbie @miltontony +1 on option 2. It's more in line with (the visual weight of) the design.

**Pull request merged**